app/poros/merchant.rb

```diff
@@ -0,0 +1,9 @@
+class Merchant
+  attr_reader :id,
+              :name
+
+  def initialize(data)
+    @id = data[:id].to_i
+    @name = data[:attributes][:name]
+  end
+end
```

Comment:
be careful naming an attribute "id", since poros automatically have a unique id associated with them. This could cause some issues when you're trying to access the id that you manually set later on.